sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -1115,13 +1115,17 @@ case class AddMonths(startDate: Expression, numMonths: Expression)
 
   override def prettyName: String = "add_months"
 }
-
 /**
- * Returns number of months between dates date1 and date2.
- */
+   * Returns number of months between dates `timestamp1` and `timestamp2`.
```

Comment:
Hm, this should have been caught by Scala linter because we follow Java style comment. See "Code documentation style" in http://spark.apache.org/contributing.html